go/otlp/client/client.go

```diff
@@ -13,15 +13,17 @@ import (
 	"context"
 	"fmt"
 	"log"
-	mathrand "math/rand"
 	"net/http"
 	"os"
 	"time"
 
+	"go.opentelemetry.io/contrib/propagators/b3"
 	"go.opentelemetry.io/otel/api/global"
-	"go.opentelemetry.io/otel/api/kv"
+	"go.opentelemetry.io/otel/api/propagation"
 	"go.opentelemetry.io/otel/exporters/otlp"
-	"go.opentelemetry.io/otel/instrumentation/httptrace"
+
```

Comment:
nit rm space (and then fix import ordering?)

Reply:
fixed